test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -108,7 +108,7 @@ public void addCommand_validData_correctlyConstructed() throws Exception {
         assertTrue(p.getPhone().isPrivate());
         assertEquals(Email.EXAMPLE, p.getEmail().value);
         assertFalse(p.getEmail().isPrivate());
-        assertEquals(Address.EXAMPLE, p.getAddress().value);
+        assertEquals(Address.EXAMPLE, p.getAddress().toString());
```

Comment:
You changed these tests to make the project compile, but you've missed out on the actual test cases (Expected and Input.txt), and the user guide.